include/quiche.h

```diff
@@ -33,6 +33,14 @@ extern "C" {
 
 #include <stdint.h>
 #include <stdbool.h>
+#include <stddef.h>
+#ifdef __unix__
+#include <sys/types.h>
```

Comment:
Why is this needed? Shouldn't `stddef.h` be enough?

Reply:
`size_t` is part of the C standard, but `ssize_t` is part of POSIX
http://man7.org/linux/man-pages/man0/stddef.h.0p.html
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_types.h.html

For POSIX systems inclusion of sys/types.h without stddef.h might be sufficient, but that might not work for Windows.